tensorflow/core/kernels/mlir_generated/base_ops_test.h

```diff
@@ -199,6 +199,16 @@ absl::InlinedVector<T, 10> DefaultInputNonZero() {
                                          0.2, 0.3, 0.5, 0.7, 0.9, 9.0, 18.0});
 }
 
+template <typename T, std::enable_if_t<
+                           llvm::is_one_of<T, Eigen::half, float, double>::value,
+                           bool> = true>
+ absl::InlinedVector<T, 10> DefaultInputVeryLarge() {
```

Comment:
I think we can actually reuse the `DefaultInputValues`. The thing we care about is only the size, not the values.

Reply:
done